packages/tools/src/utilities/calibrateImageSpacing.ts

```diff
@@ -15,19 +15,17 @@ const { calibratedPixelSpacingMetadataProvider } = utilities;
 export default function calibrateImageSpacing(
   imageId: string,
   renderingEngine: Types.IRenderingEngine,
-  rowPixelSpacing: number,
-  columnPixelSpacing: number
+  spacing: Types.IImageCalibration | number
 ): void {
-  // 1. Add the calibratedPixelSpacing metadata to the metadata provider
-  // If no column spacing provided, assume square pixels
-  if (!columnPixelSpacing) {
-    columnPixelSpacing = rowPixelSpacing;
+  // Handle simple parameter version
+  if (typeof spacing === 'number') {
+    spacing = {
+      type: Enums.CalibrationTypes.USER,
+      scale: spacing,
```

Comment:
`scale : spacing` does not makes sense to me at all. Why we are saying scale is equal to spacing? Shouldn't it be the division?

Reply:
It doesn't work with the old values anyways, so I'm removing that code.